buildpacks/buildpack-ruby/tests/ruby-sinatra/Gemfile.lock

```diff
@@ -7,7 +7,7 @@ GEM
       rack
     rack-test (0.6.3)
       rack (>= 1.0)
-    rake (12.0.0)
+    rake (12.3.2)
```

Comment:
Minor change, use system available rake instead of installing old one (outdated)